src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -141,31 +141,31 @@ public static void main(String[] args) {
         EventQueue.invokeLater(new Runnable() {
             public void run() {
                 Application app = new Application("Jobs 2D");
-                DrawerFeature.setupDrawerPlugin(app);
-                CommandsFeature.setupCommandManager();
+                DrawerFeature.getInstance().setupDriverFeature(app, null);
+                CommandsFeature.getInstance().setupDriverFeature(app, null);
 
                 List<ICanvas> canvases = new ArrayList<>();
                 for (Format format : Format.values()) {
                     canvases.add(new RectangleCanvas(format));
                 }
                 canvases.add(new RectangleCanvas(200, 100, "Rectangle 200x100", "custom"));
                 canvases.add(new EllipseCanvas(150, 100, "Ellipse rx:150 ry:100", "custom"));
-
+                
                 CanvasFeature.setCanvases(canvases);
-                CanvasFeature.setupCanvasFeature(app);
+                CanvasFeature.getInstance().setupDriverFeature(app, null);
 
-                UsageMonitorFeature.setupDeviceMonitorPlugin(DriverFeature.getDriverManager());
-                DriverFeature.setupDriverPlugin(app);
+                UsageMonitorFeature.getInstance().setupDriverFeature(null,DriverFeature.getDriverManager());
+                DriverFeature.getInstance().setupDriverFeature(app, DriverFeature.getDriverManager());
                 setupDrivers(app);
-                TransformationFeature.setupTransformationPlugin(app, DriverFeature.getDriverManager());
-                MacroFeature.setupMacroFeature(app, DriverFeature.getDriverManager());
+                TransformationFeature.getInstance().setupDriverFeature(app, DriverFeature.getDriverManager());
+                MacroFeature.getInstance().setupDriverFeature(app, DriverFeature.getDriverManager());
```

Comment:
Code duplication (e.g., getInstance().setupDriverFeature). Features management should simplify configuring the application. e.g., features should be added to FeatureManager class and configured all at once.